usuarios/delete-usuarios.spec.js

```diff
@@ -0,0 +1,98 @@
+const faker = require('faker')
+const testServer = require('../utils/testServer')
+const { postUser } = require('../utils/user')
+const { getAdminUserById } = require('../utils/user')
+
+const rotaUsuarios = '/usuarios'
+const rotaLogin = '/login'
+const rotaProdutos = '/produtos'
+const rotaCarrinhos = '/carrinhos'
```

Comment:
extrair isso aqui pra um arquivo de utils chamado rotas.js
`module.exports = {
    rotaUsuarios: '/usuarios',
    rotaLogin: '/login',
    rotaProdutos: '/produtos',
    rotaCarrinhos: '/carrinhos'
}`

Reply:
ai com a criação desse arquivo, isso aqui vira 
```suggestion
const { rotaUsuarios, rotaLogin, rotaProdutos, rotaCarrinhos} = require('../utils/rotas')
```